### PR TITLE
change

### PR DESCRIPTION
**Pull request by yuzp1996**

### CODE REVIEW

**Comment by yuzp1996** [comment]
This PR is not need merge

File: plugin/client/gitpullrequest.go [hunk 1]
```diff
@@ -55,6 +54,10 @@ func newGitPullRequest(client Client, meta Meta, secret corev1.Secret) ClientGit
 func (g *gitPullRequest) Create(ctx context.Context, baseURL *duckv1.Addressable, payload metav1alpha1.CreatePullRequestPayload, options ...OptionFunc) (*metav1alpha1.GitPullRequest, error) {
 	prObj := &metav1alpha1.GitPullRequest{}
 	options = append(options, MetaOpts(g.meta), SecretOpts(g.secret), BodyOpts(payload), ResultOpts(prObj))
+	// 这里获取 repository 的方式 可能需要调整
+	// 目前 payload.Source.project 和  payload.Source.repository 是分开的  比如 Project 是 yuzp1996 repository是 gitlabzpyu   对应的就是这个项目 https://gitlab.com/yuzp1996/gitlabzpyu
+	// 而且我那边只能获取到 https://gitlab.com/yuzp1996/gitlabzpyu 这个信息  我是拿不到 repoID 的   所以需要这边做一个转换
```
Comment (danielfbm) [inline plugin/client/gitpullrequest.go:59]: 英文注释

Comment (danielfbm) [reply plugin/client/gitpullrequest.go:59]: 避免留下存在的账号信息

**Review by danielfbm — CHANGES_REQUESTED** [review]
PR title改成一个具体的change吧，否则搜索等等都没法做

**Comment by yuzp1996** [comment]
Sorry for open this PR  maybe I should use another way to indicate my change which I want others to make 

I  will close this PR right now

**Pull request closed without merging**